src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge54.java

```diff
@@ -0,0 +1,15 @@
+package org.owasp.wrongsecrets.challenges;
+
+import org.springframework.stereotype.Component;
+
+/** Challenge with a secret in .gitignore */
+@Component
+public class Challenge54 extends FixedAnswerChallenge {
+
+  private final String secret = "(<:GITIGN0RE_SECRET:>)";
```

Comment:
Lets have the encrypted value here and match plaintext entries from users after encryption